<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>253</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fontComboBox</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>Form</receiver>
   <slot>change_to_new_font(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>change_to_new_font(QFont)</slot>
 </slots>
</ui>
